tests/menubar-editor/commands.json

```diff
@@ -0,0 +1 @@
+{"navigate to menuitemcheckbox in submenu using interaction mode": {"interaction": {"jaws": [["ENTER"], ["SPACE"], ["DOWN"]]}}, "close submenu of menubar using reading mode": {"reading": {"jaws": [["ESC"]]}}, "open submenu of menubar using interaction mode": {"interaction": {"jaws": [["ENTER"], ["SPACE"], ["UP_AND_DOWN"]]}}, "read menu item in menubar using interaction mode": {"interaction": {"jaws": [["INS_TAB"], ["INS_UP"]]}}, "read disabled menuitem in a submenu using interaction mode": {"interaction": {"jaws": [["INS_TAB"], ["INS_UP"]]}}, "tab to menubar in reading mode": {"reading": {"jaws": [["TAB_AND_SHIFT_TAB"]]}}, "change state of menuitemcheckbox in submenu using interaction mode": {"interaction": {"jaws": [["ENTER"], ["SPACE"]]}}, "navigate to menubar in reading mode": {"reading": {"jaws": [["UP_AND_DOWN"], ["F_AND_SHIFT_F"]]}}, "read menuitemcheckbox in a submenu using reading mode": {"reading": {"jaws": [["INS_TAB"], ["INS_UP"]]}}, "navigate to menuitemcheckbox in submenu using reading mode": {"reading": {"jaws": [["UP_AND_DOWN"]]}}, "change state of menuitemradio in submenu using interaction mode": {"interaction": {"jaws": [["ENTER"], ["SPACE"]]}}, "close submenu of menubar using interaction mode": {"interaction": {"jaws": [["ESC"]]}}, "read disabled menuitem in a submenu using reading mode": {"reading": {"jaws": [["INS_TAB"], ["INS_UP"]]}}, "navigate to menuitem in menubar using interaction mode": {"interaction": {"jaws": [["TAB_AND_SHIFT_TAB"]]}}, "read menuitemradio in a group in submenu using reading mode": {"reading": {"jaws": [["INS_TAB"], ["INS_UP"]]}}, "read menuitemradio in a group in a submenu using interaction mode": {"interaction": {"jaws": [["INS_TAB"], ["INS_UP"]]}}, "read menuitemcheckbox in a submenu using interaction mode": {"interaction": {"jaws": [["INS_TAB"], ["INS_UP"]]}}, "change state of menuitemcheckbox in submenu using reading mode": {"reading": {"jaws": [["ENTER"], ["SPACE"]]}}, "read menu item in menubar using reading mode": {"reading": {"jaws": [["INS_TAB"], ["INS_UP"]]}}, "navigate to menuitemradio in submenu using interaction mode": {"interaction": {"jaws": [["ENTER"], ["SPACE"], ["DOWN"]]}}, "navigate to menuitem in menubar using reading mode": {"reading": {"jaws": [["LEFT_AND_RIGHT", " (with Smart Navigation on)"]]}}, "change state of menuitemradio in submenu using reading mode": {"reading": {"jaws": [["ENTER"], ["SPACE"]]}}, "navigate to menuitemradio in submenu using reading mode": {"reading": {"jaws": [["UP_AND_DOWN"]]}}, "open submenu of menubar using reading mode": {"reading": {"jaws": [["ENTER"], ["SPACE"]]}}}
```

Comment:
It could, but I did have the suggestion/thought from Mike that if it is printed like this then people might realize it is generated and not for manual editing.

Reply:
It wasn't clear to me. Is there a way for us to note this difference?

Reply:
very soon all of these files will be generated, so there won't be a need for the distinction!